### PR TITLE
add c sharp samples to step_by_step/resources

### PR DESCRIPTION
**Pull request by KellyThomas**

### CODE REVIEW

File: getting_started/step_by_step/resources.rst [hunk 1]
```diff
@@ -82,21 +82,36 @@ Loading resources from code
...
 
     func _ready():
             var res = load("res://robi.png") # resource is loaded when line is executed
             get_node("sprite").texture = res
 
+ .. code-tab:: csharp
+
+    public override void _Ready()
+    {
+        var texture = (Texture) GD.Load("res://robi.png"); // resource is loaded when line is executed
```
Comment (paulloz) [inline getting_started/step_by_step/resources.rst]: Classic nitpicking: there should be no space between `(Texture)` and `GD`.  
Same on the next line.

File: getting_started/step_by_step/resources.rst [hunk 2]
```diff
@@ -107,12 +122,23 @@ To obtain an instance of the scene, the method
...
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _on_shoot():
             var bullet = preload("res://bullet.tscn").instance()
             add_child(bullet)                  
 
+ .. code-tab:: csharp
+
+    PackedScene _bulletScene = (PackedScene) GD.Load("res://bullet.tscn");
```
Comment (paulloz) [inline getting_started/step_by_step/resources.rst]: Same thing for the cast.  
Also if this field could be explicitly `private` that'd be great.

**Comment by KellyThomas** [comment]
Thanks @paulloz, both taken care of now.

**Comment by mhilbrunner** [comment]
Merging. Thanks for the big batch of C# PRs :)

**Pull request merged**